### ISSUE TITLE
Wiki: Additional information about music videos moved from old channels to new channels

### ISSUE THREAD

**Issue by tech234a** [mattwright324/youtube-metadata#173]
Just noting that on Windows, if you use your the volume keys or the play/pause button on your keyboard during one of these videos, you will receive a media status pop-up in the corner of your screen containing the _old_ channel name along with the video title and thumbnail. I've noticed this with a lot of music videos.

**Comment by mattwright324** [comment]
I didn't understand the media keys part but when the video hud is visible (such as when paused) there is an info icon in the top right that seems to display the old channel name as you are describing. Updated that entry in the wiki to note this or if I understood wrong let me know thanks!

**Comment by tech234a** [comment]
Yeah the little music info icon in Chrome would display the same information as is displayed when using the keyboard play/pause/volume keys.

**Comment by tech234a** [comment]
Oh looking at the update you made it looks like you found something different than I was referring to.

I was referring to the information displayed by: https://web.dev/articles/media-session

**Comment by mattwright324** [comment]
I'm using FireFox so not sure if that is part of my issue.

**Comment by codex1001** [comment]
> Just noting that on Windows, if you use your the volume keys or the play/pause button on your keyboard during one of these videos, you will receive a media status pop-up in the corner of your screen containing the _old_ channel name along with the video title and thumbnail. I've noticed this with a lot of music videos.

Hello, can you give an example URL of the videos you mentioned and add a screenshot?
